Matt's branch

mattmanb opened the pull request:
Chance/Community cards added

mattmanb committed:
added startAuction function

mattmanb committed:
Merge branch 'master' into Matt's-Branch

mattmanb committed:
Started work on auctions (not finished yet);
added in 'modals' for buying properties so alerts are no longer used;
using modals for the auction should fix my problem, which I will implement later;

mattmanb committed:
Implemented chance_deck and community_deck classes
and represented them in board.js. Still need to add
functionality in landOn function in board.js. Still
working on the special_event class and auctioning.

mattmanb committed:
Merge branch 'master' into Matt's-Branch

mattmanb committed:
Pieces get loaded in and mapped to the board.

mattmanb committed:
Pieces can move all the way around the board now!

mattmanb committed:
Lots of chat changed - added a game event chatbot
that tells the player exactly what is happening

mattmanb committed:
Chance and community cards implemented
through card and card_deck

Co-authored-by: carloub <carloub@users.noreply.github.com>

mattmanb committed:
imported card.js and card_deck.js into backend.js

JavierCrespoLaw reviewed (commented):
Board.js still requires chance and community cards, which don't exist anymore. Other than that, everything is good.

mattmanb committed:
Various bug fixes across the project; auctions
work now

mattmanb committed:
Added a display that shows players' info
like name, money, if they're in jail, etc.

Also made the background of the player whose turn
it is lime green, and your player's info box has
a text color of red.

JavierCrespoLaw reviewed (approved):
Nice!
